doc/handle_scope.md

```diff
@@ -34,7 +34,7 @@ Napi::HandleScope::HandleScope(Napi::Env env, Napi::HandleScope scope);
 - `[in] scope`: pre-existing `Napi::HandleScope`.
 
 Returns a new `Napi::HandleScope` instance which wraps the napi_handle_scope
```

Comment:
Could you use backtick for napi_handle_scope? From napi_handle_scope to `napi_handle_scope`